### PR TITLE
V3.0.0

### PR DESCRIPTION
**Pull request by notlee**
- Remove the ability to customise the base class name `o-layout`: https://github.com/Financial-Times/o-layout/pull/40
- Remove o-header-services and o-footer-services as dependancies: https://github.com/Financial-Times/o-layout/pull/41
- Use `o-grid` for container size and media queries: https://github.com/Financial-Times/o-layout/pull/42
- Do not set `$o-brand`: https://github.com/Financial-Times/o-layout/pull/44
- Make many existing mixins private: https://github.com/Financial-Times/o-layout/pull/45
-  Adds support for a landing page layout, with overview and hero areas: https://github.com/Financial-Times/o-layout/pull/46, https://github.com/Financial-Times/o-layout/pull/47
- Adds support for a query page layout, with heading, query-sidebar, and query-aside areas: https://github.com/Financial-Times/o-layout/pull/48
- Adds class `o-layout-typography` to share typography styles: https://github.com/Financial-Times/o-layout/pull/49
- Moves to floats from CSS grid for the main content area to enable collapsing margins and to avoid the [Firefox bug](https://github.com/Financial-Times/o-layout/issues/50), the [Chrome bug](https://github.com/Financial-Times/o-layout/issues/36), and [hack mchack](https://github.com/Financial-Times/o-layout/blob/v2.2.4/src/scss/_grid-areas.scss#L69): https://github.com/Financial-Times/o-layout/pull/52
- Decouple the navigation from heading links. Linkable headers are useful outside of the documentation layout. Throttles navigation highlighting on scroll for performance. Tweaks heading links octothorpe appearance. https://github.com/Financial-Times/o-layout/pull/53
- Support for a navigation with hierarchy. https://github.com/Financial-Times/o-layout/pull/68
- Support the navigation within the query layout. https://github.com/Financial-Times/o-layout/pull/67
- Makes the sidebar nav scrollable. https://github.com/Financial-Times/o-layout/pull/70

**Documentation layout**
screenshot 2019-02-07 at 11 14 42

**Query layout**
screenshot 2019-02-07 at 11 14 51

**Landing layout**
screenshot 2019-02-07 at 11 14 38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,242 +1,446 @@
...
+[See the registry demos](https://registry.origami.ft.com/components/o-layout#demo-landing-layout) for an example landing page.
+
+## Query Layout
+
+The query layout is intended for search, filter, and result pages. As well as the header and footer, the query layout provides four areas:
 
-	<footer class="o-layout__footer">
-		<!-- o-footer-services markup goes here -->
-	</footer>
+- Heading
```
[the remark this answers is not in the document]
Comment (notlee) [reply README.md]: It is a little confusing but I don't like page title as you might have more in that section. It's a "should come before query/search inputs" section.

File: README.md [hunk 2]
```diff
@@ -1,242 +1,446 @@
...
+| MAIN CONTENT       |
+├————————————————————┤
+| ASIDE SIDE BAR     |
+├————————————————————┤
+|       FOOTER       |
+└————————————————————┘
+```
+
+## Sass
+As with all Origami components, o-layout has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). You can include o-layout styles with the `oLayout` mixin.
```
[the remark this answers is not in the document]
Comment (notlee) [reply README.md]: I've been mentioning it briefly until it's officially dead, but think your approach is better.
```suggestion
You can include o-layout styles with the `oLayout` mixin.
```

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gvonkoss) [reply README.md]: complic8d

**Comment by notlee** [comment]
:shipit:

**This was shipped as beta.2**: Commits following will be released in future betas.

**Comment by notlee** [comment]
:shipit:

**This was shipped as beta.3**: Commits following will be released in future betas.

**Comment by notlee** [comment]
:shipit:

**This was shipped as beta.4**: Commits following will be released in future betas.

**Pull request merged**